src/app/Main/Body/QueryPage/Drawer/Displayed_Features/Feature-SelectorElement.component.tsx

```diff
@@ -34,7 +34,6 @@ export function DisplayedFeatureSelectorComponent(
 
     return (
         <Select
-            key={props.keyValue}

```

Comment:
Reject: I think the keyValue can also be removed from FeatureSelectorProps interface

Reply:
I removed it in the commit "Add description to functions", because I noticed that I no longer need the key. Sorry for mixing it up.

Reply:
no problem, this is will happen sometimes, you can just comment that you fixed it in another commit